nflow-engine/src/main/resources/scripts/db/h2.create.ddl.sql

```diff
@@ -11,13 +11,13 @@ create table if not exists nflow_workflow (
   external_id varchar(64) not null,
   state varchar(64) not null,
   state_text varchar(128),
-  next_activation timestamp,
-  external_next_activation timestamp,
+  next_activation timestamp with time zone,
+  external_next_activation timestamp with time zone,
   executor_id int,
   retries int not null default 0,
-  created timestamp not null default current_timestamp,
-  modified timestamp not null default current_timestamp,
-  started timestamp,
+  created timestamp with time zone not null default current_timestamp,
```

Comment:
Should there be any changes to scripts that change existing h2 databases?

Reply:
hmm. Maybe not. But I'll add an entry to changes that says if you are using h2 as real database you must backup + recreate + restore backup. That is anyway the only h2 1.x to 2.x upgrade procedure.